### PR TITLE
RNs-23-feb-23

### PR DESCRIPTION
**Pull request by janettelynch**

### CODE REVIEW

File: knowledgeBase/release-notes/2023-Feb-2.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
 # JupiterOne 16 February 2023 Release
 
 ## New Features and Improvements
--   
+-   Search for integration instances to use in certain Alert Actions, making the process of creating and configuring a new Alert`Rule` straightforward as you are now able to search directly for the integration source you want to use.
```
Comment (janettelynch) [inline knowledgeBase/release-notes/2023-Feb-2.md:4]: @sarabeth I don't really understand line 4. Could you reword another way?

Comment (janettelynch) [reply knowledgeBase/release-notes/2023-Feb-2.md:4]: Can you dumb it down to the steps you are taking in the search.

Comment (sarabeth) [reply knowledgeBase/release-notes/2023-Feb-2.md:4]: When setting up an Alert Rule, you can now search for a specific integration instance to use with an Action.

**Review by AnneNielsenJ1 — APPROVED** [review]
